File: VB/CustomTokenEdit/CustomTokenEditSelectedItemCollection.vb
Imports DevExpress.XtraEditors
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraEditors.Repository
Namespace Example.CustomTokenEdit
	Public Class CustomTokenEditSelectedItemCollection
		Inherits TokenEditSelectedItemCollection

		Public Sub New(ByVal properties As RepositoryItemTokenEdit)
			MyBase.New(properties)
		End Sub

		Default Public ReadOnly Property Item(ByVal index As Integer) As CustomTokenEditToken
			Get
				Return DirectCast(InnerList(index), CustomTokenEditToken)
			End Get
		End Property
	End Class
End Namespace

File: CS/TokenEdit_Glyph_InEndOfToken/CustomTokenEdit/CustomTokenEditSelectedItemCollection.cs
using DevExpress.XtraEditors;
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraEditors.Repository;
namespace TokenEdit_Glyph_InEndOfToken.CustomTokenEdit
{
    public class CustomTokenEditSelectedItemCollection : TokenEditSelectedItemCollection
    {
        public CustomTokenEditSelectedItemCollection(RepositoryItemTokenEdit properties) : base(properties) { }

        public new CustomTokenEditToken this[int index] { get { return (CustomTokenEditToken)InnerList[index]; } }
    }
}
